bokeh/core/has_props.py

```diff
@@ -145,18 +145,18 @@ def __new__(meta_cls, class_name, bases, class_dict):
 
         return super().__new__(meta_cls, class_name, bases, class_dict)
 
-    def __init__(cls, _, bases, __):
+    def __init__(cls, class_name, bases, __):
         # Check for improperly redeclaring a Property attribute.
         for base in bases:
             if not issubclass(base, HasProps):
                 continue
             base_properties = base.properties()
             for attr in cls.__dict__: # we do NOT want inherited attrs here
                 if attr in base_properties:
-                    warn("Property {attr!r} in class {base.__name__} was redeclares by a class attribute "
-                         "{attr!r} in class {class_name}; it never makes sense to do this. "
-                          "Either {base.__name__}.{attr} or {class_name}.{attr} should be removed,"
-                          "or Override() should be used to change a default value of a base class property.",
+                    warn(f"Property {attr!r} in class {base.__name__} was redeclared by a class attribute "
+                         f"{attr!r} in class {class_name}; it never makes sense to do this. "
+                         f"Either {base.__name__}.{attr} or {class_name}.{attr} should be removed, "
```

Comment:
BTW is an `f` prefix required on every line here? I'm fine merging as-is but curious whether the first `f` applies across implicit concatenations.

Reply:
Yes, but I needed to test it to be 100% sure.

![image](https://user-images.githubusercontent.com/19758978/111701508-3e9f3000-883b-11eb-9e6d-d881b87aade2.png)

Reply:
TBH that seems surprising, if it was an intentional decision!